src/main/java/com/psipher/application/actions/UserDetailsOperationsImpl.java

```diff
@@ -125,6 +125,43 @@ public String deleteUser(String userId) throws DDBException {
         return status;
     }
 
+    @Override
+    public String deleteUserAccount(String userId, String domain, String userAccount) throws DDBException {
+        UserDDBModel userDDBModel = viewDetails(userId);
+        List<WebsiteDDBModel> websiteDDBModels = userDDBModel.getWebsites();
+        for (WebsiteDDBModel websiteDDBModel : websiteDDBModels) {
```

Comment:
This is a redundant code which can be refractored. This code is also been used by deletedomain function. Try to move to single function

Reply:
it will make code more complex